.github/workflows/regression-report.yml

```diff
@@ -222,10 +225,58 @@ jobs:
         run: |
           ./.ci/append-to-github-output.sh "message" "$(cat .ci-temp/message)"
 
+  check_cancellation:
+    needs: [parse_comment, make_report]
+    if: always()
+    runs-on: ubuntu-latest
+    outputs:
+      cancelled: ${{ steps.check_status.outputs.cancelled }}
+    steps:
+      - name: Check job status
+        id: check_status
+        run: |
+          if [ "${{ contains(needs.*.result, 'cancelled') }}" = "true" ]; then
+            echo "Job was cancelled due to concurrency"
+            echo "cancelled=true" >> $GITHUB_OUTPUT
+          else
+            echo "Job was not cancelled"
+            echo "cancelled=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Change reaction if cancelled
+        if: steps.check_status.outputs.cancelled == 'true'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            // Remove the rocket reaction
```

Comment:
Let's not do this, it is too fancy, let's keep it simple, it will be more robust.
Just add sad emoji.

Reply:
Two emoji will mean: it is started and failed/canceled

Reply:
> Two emoji will mean: it is started and failed/canceled

@romani 
![Uploading image.png…]()